src/backend/marsha/core/tests/test_serializers_update_state.py

```diff
@@ -113,9 +113,32 @@ def test_serializers_update_state_key_elements(self):
         self.assertEqual(
             serializer.get_key_elements(),
             {
+                "extension": None,
                 "model_name": "video",
                 "object_id": str(object_id),
                 "stamp": "1533686400",
                 "uploaded_on": datetime(2018, 8, 8, tzinfo=pytz.utc),
             },
         )
+
+    def test_serializers_update_state_key_elements_with_extension(self):
+        """Test the presence of an extension."""
```

Comment:
It's not really clear in the docstring what you are trying to prove. I find it user when we write something of the form "the serializer should extract extension from the key parameter"?